### PR TITLE
Shifted position of progressbar in Upload activity please wait screen

### LINKED ISSUE #2329: Center vertically "Receiving shared content"
**UI of Upload Activity can be improved** 

Currently the Ui has a progress bar is at the bottom with a text view which is nearly aligned with the bottom. leaving the whole activity blank:
whatsapp image 2019-01-22 at 10 05 03 am

How can we reproduce the issue? 
Select the FAB from home screen  then upload an image You will see the above screen for a few seconds.

**What could have made it better**
We can use material components or a custom ProgressBar in the middle of the screen like the one given below to improve the current UI.
1tkwg

_Suggestions are most welcome_ :)
I'll start working on this immediately after the UI is selected.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by vanshikaarora**
**Modified Upload Activity**

Fixes #2329 Center vertically "Receiving shared content"

**What changes did you make and why?**
Modified upload activity and it's view flipper to set content view as activity_upload_please_wait in place of the view flipper at the bottom

**Screenshots showing what changed (optional - for UI changes)**

whatsapp image 2019-01-23 at 10 20 52 pm

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -457,6 +457,7 @@ Upload your first media by touching the camera or gallery icon above.</string>
     <item quantity="other">Receiving shared content. Processing the images might take some time depending on the size of the images and your device</item>
   </plurals>
 
+  <string name="recieving_content">Receiving shared content. Processing the image might take some time depending on the size of the image and your device</string>
```
Comment (maskaravivek) [inline app/src/main/res/values/strings.xml]: This string should already exists. `receiving_shared_content `

File: app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java [hunk 2]
```diff
@@ -321,15 +321,17 @@ public void setRightCardVisibility(boolean visible) {
     @Override
     public void setBottomCardVisibility(@UploadPage int page, int uploadCount) {
         if (page == TITLE_CARD) {
+            setContentView(R.layout.activity_upload);
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: why is this required?

**Comment by maskaravivek** [comment]
@vanshikaarora Can you rename the title of the PR to describe the issue more appropriately.

**Comment by vanshikaarora** [comment]
> @vanshikaarora Can you rename the title of the PR to describe the issue more appropriately.

@maskaravivek  I'll do that

**Comment by maskaravivek** [comment]
@vanshikaarora Can you address the comments in this PR so that we can merge it.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (vanshikaarora) [reply app/src/main/res/values/strings.xml]: Yes I'll remove that

File: app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java (hunk 2 above)
Comment (vanshikaarora) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: At the very first time when value of page will be 0(i.e PLEASE_WAIT) thus it will set the contentview to R.layout.activity_upload_please_wait the next time page comes with any value we need to first set the content view to R.layout.activity_upload and then use the view flipper. This was the logic over which I had worked upon. But curently I think there can be a better solution than this.

Can you please give some more time to come up with a better solution

Comment (maskaravivek) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: `activity_upload_please_wait` is also part of the `ViewFlipper`. You basically need to hide the other elements ie `activity_upload_top_card` and `activity_upload_right_card` so that the ViewFlipper can take up the whole space. 

Infact, if you go through the upload flow, you would notice that  `activity_upload_top_card` and `activity_upload_right_card` are visible only along with `activity_upload_bottom_card`. It would be logical to combine them as a single child of the `ViewFlipper`. That way the `ViewFlipper` can always `match_parent`.

Comment (vanshikaarora) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: I have made the following changes:
Changed attribute of ViewFlipper to match-parent
copied the content of `activity_upload_top_card` and `activity_upload_right_card` to `activity_upload_bottom_card`
Kindly review and share feedback's :)

Comment (maskaravivek) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: It looks like you forgot to push your changes.

Comment (vanshikaarora) [reply app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java]: I am sorry. Actually I have changed the branch, that was the problem. I have send another PR here https://github.com/commons-app/apps-android-commons/pull/2370
